v1/submissions.go

```diff
@@ -144,6 +145,229 @@ type Submission struct {
 	CloseReason string
 }
 
+/*
+	INTIGRITI Report structure
```

Comment:
I think we can remove this comment, and for reference to enable Go Doc, comments should follow this standard: https://tip.golang.org/doc/comment but I do not think it is necessary here